integration/test/src/org/broadinstitute/reaper/server/IntegrationContainersTest.scala

```diff
@@ -0,0 +1,150 @@
+      failures => throw new RuntimeException(s"Could not load Reaper config: $failures"),
+      config => config
+    )
+
+  /** URL of vault server to use when getting bearer tokens for service accounts. */
+  private val vaultUrl = "https://clotho.broadinstitute.org:8200/"
+
+  /** Path in vault to the service account JSON to use in testing. */
+  private val vaultPath = "secret/dsde/gotc/test/clio/clio-account.json"
+
+  /** List of possible token-file locations, in order of preference. */
+  private val vaultTokenPaths = Seq(
+    File("/etc/vault-token-dsde"),
+    File(System.getProperty("user.home"), ".vault-token")
+  )
+
+  /** Pull service account credentials to use when accessing cloud resources from Vault. */
+  protected def loadCredentials(): GoogleCredentials = {
+    val vaultToken: String = vaultTokenPaths
+      .find(_.exists)
+      .map(_.contentAsString.stripLineEnd)
+      .getOrElse(sys.error("Vault token not found on filesystem!"))
+
+    val vaultConfig = new VaultConfig()
+      .address(vaultUrl)
+      .token(vaultToken)
+      .build()
+
+    val vaultDriver = new Vault(vaultConfig)
+    val accountJSON =
+      vaultDriver
+        .logical()
+        .read(vaultPath)
+        .getData
+        .asScala
+        .toMap[String, String]
+        .asJson
+
+    val jsonStream = new ByteArrayInputStream(
+      accountJSON.pretty(Printer.noSpaces.copy(dropNullValues = true)).getBytes()
+    )
+    ServiceAccountCredentials.fromStream(jsonStream)
+  }
+
+  private val reaperContainer = ReaperContainer.container
+  private val cromwellContainer = CromwellContainer.container
+  private val clioContainer: ClioDockerComposeContainer =
+    ClioDockerComposeContainer.waitForReadyLog(File("/tmp"))
+  private val postgresContainer = PostgreSQLContainer()
+
+  private lazy val db = Database.forURL(
+    s"jdbc:postgresql://${postgresContainer.containerIpAddress}:${postgresContainer.mappedPort(5432)}/test",
+    driver = "org.postgresql.Driver",
+    user = "test",
+    password = "test"
+  )
+
+  private lazy val clioWebClient: ClioWebClient =
+    ClioWebClient(
+      clioCredentials,
+      clioContainer.clioHost,
+      clioContainer.clioPort,
+      useHttps = false
+    )
+
+  private lazy val clioCredentials = new ClioCredentials(loadCredentials())
+
+  "IntegrationContainersTest" should "be able to connect and query postgres db" in {
+    assert(
+      db.createSession().createStatement().execute(postgresContainer.testQueryString)
+    )
+  }
+
+  "IntegrationContainersTest" should "be able to get the current running clio version" in {
+    // Will need some way to get this from build
+    clioWebClient.getClioServerVersion.runWith(Sink.head).map { json =>
+      json.as[VersionInfo] should be(
+        Right(VersionInfo("d695feea7a50c073bf3764fd329f20e9e49d515d"))
+      )
+    }
+  }
+
+  "IntegrationContainersTest" should "be able to get the current running reaper version" in {
+    val response = requests.get(
```

Comment:
Where is `requests` coming from?

Reply:
requests coming from 
https://github.com/broadinstitute/reaper/pull/6/files#diff-2a181b07f2da6a3a4e2aa5e6ad907a9eR73

https://github.com/lihaoyi/requests-scala

Reply:
mill puts it on the classpath just like any other jar.. the thing that I think is confusing is that requests is a package object at the root level so it can be referenced directly

Reply:
Do you think it would be less confusing to do.

```
import requests._

val response = get(...)
```

Reply:
Probably not, then I'd be confused about where `get` comes from...

Reply:
I think the thing that's confusing is that `requests` is global and I'm never expecting to see that.